meta/WriteOut.m

```diff
@@ -1225,8 +1225,20 @@
  */
 void " <> modelName <> "_slha_io::set_decays(const " <> modelName <> "_decay_table& decay_table, FlexibleDecay_settings const& flexibledecay_settings)
 {
+   auto is_higgs =
+      [](int id) {
+         bool is_whatever_higgs = false;
+         for (auto candidate : {25, 35, 37}) {
```

Comment:
Sorry to be too pedantic, but I think this is an excellent candidate for `std::any_of`. :) We could write something along the lines
~~~.cpp
const int pids[] = { 25, 35, 37 }; // Higgs particle IDs
return std::any_of(std::begin(pids), std::end(pids), [&](){ /* predicate here */ });
~~~